src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition:
+    transform 300ms ease,
+    background-color 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-family: $font-family;
+    color: #060b35;
+    margin-top: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    transition: color 300ms ease;
+
+    .card:hover & {
+      color: #34568b;
+    }
```

Comment:
The hover effect for changing the color of `.card__title` is applied when hovering over the `.card` element. This could lead to unexpected behavior if there are other elements inside `.card` that also change on hover. Consider moving the hover effect to `.card__title:hover`.